tests/sentry/integrations/gitlab/test_client.py

```diff
@@ -122,3 +124,34 @@ def test_no_refresh_when_api_call_successful(self):
         self.assert_response_call(call, self.request_url, 200)
         assert resp == self.request_data
         self.assert_identity_was_not_refreshed()
+
+    @responses.activate
+    def test_check_file(self):
+        path = "file.py"
+        ref = "537f2e94fbc489b2564ca3d6a5f0bd9afa38c3c3"
+        responses.add(
+            responses.HEAD,
+            "https://example.gitlab.com/api/v4/projects/{}/repository/files/{}?ref={}".format(
+                self.gitlab_id, path, ref
+            ),
+            json={"text": 200},
+        )
+
+        resp = self.client.check_file(self.gitlab_id, path, ref)
+        assert responses.calls[0].response.status_code == 200
+        assert resp == 200
```

Comment:
@ceorourke I'm guessing this part fails now, yea?

Reply:
@scefali no, it passes. `check_file` returns the status code.